spec/models/course_user_spec.rb

```diff
@@ -15,6 +15,14 @@
 
     it { is_expected.to be_student }
     it { is_expected.not_to be_phantom }
+
+    context 'when a user is provided' do
+      let(:user) { create(:user) }
+      subject { CourseUser.new(user: user) }
```

Comment:
that's not what the context says, the context and subject should be related. we're still talking about the `CourseUser` object.